botany_importer.py

```diff
@@ -38,8 +43,22 @@ def __init__(self, paths, config):
         #     pickle.dump(self.barcode_map, outfile)
         # else:
         #     self.barcode_map = pickle.load(open(FILENAME, "rb"))
+
+        if config == botany_importer_config:
```

Comment:
since picturae_importer and botan_importer both use the same underlying importer, this if statement is to prevent the botany_importer_config from overwriting the monitoring html while picturae_importer is running.